assets/js/components/UI/Form/BatchFieldArray.jsx

```diff
@@ -23,7 +23,7 @@ const UIFormBatchFieldArray = ({
   const [isRemove, setIsRemove] = useState();
 
   function handleAddClick() {
-    append({ metadataItem: "" });
+    append({ metadataItem: defaultValue });
```

Comment:
This is a regression which we don't want implemented.  From this PR:  https://github.com/nulib/meadow/pull/1462/files